### ISSUE TITLE
U6 ADA Issues

### ISSUE THREAD

**Issue by lmock** [bjc-edc-2017-18/bjc-r#17]
- [x] L1P2 Missing Alt Text Line 44
`<img class="indent" src="http://images.huffingtonpost.com/2016-12-19-1482152530-8466162-clintonalien630x354-thumb.jpg" alt="" title="" />`
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/1-abstraction/02-software-applications.html

- [x] L1P3 Missing Alt Text Line 31
`<img class="imageRight" src="/bjc-r/img/6-computers/language-continuum.png" alt="" title="" />
`https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/1-abstraction/03-software-languages.html

- [x] L1P3 Missing Alt Text Line 91
`<img class="inline" src="/bjc-r/img/6-computers/5-times-blank-plus-7.png" alt="" title="" />
`https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/1-abstraction/03-software-languages.html

- [x] L1P9 Truth tables missing summary/captions, scopes, and headings. 
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/1-abstraction/09-digital-logic-gates.html

- [x] L2P1 Table is missing summary, scope, headings
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/2-representation-compression/1-bits.html

- [x] L2P2 Tables are missing summary, scope, headings
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/2-representation-compression/2-sequences.html

- [x] L2P5 Tables are missing summary, scope, headings
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/2-representation-compression/5-binary.html

- [x] L2P6 Tables are missing summary, scope, headings
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/2-representation-compression/6-hexadecimal.html

- [x] L2P7 Table is missing summary, scope, headings
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/2-representation-compression/7-more-hex.html

- [x] L2P8 Table is missing summary and headings
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/6-computers/2-representation-compression/8-compression.html

**Comment by maryfries** [comment]
L1P2 Missing Alt Text Line 44: these images aren't in use. They are in a class="todo" which don't appear for students.

**Comment by maryfries** [comment]
L1P9 Truth tables missing summary/captions, scopes, and headings: I added summaries and scopes, but there are already headings here.

**Comment by maryfries** [comment]
L2P1, L2P2, L2P7 Table is missing summary, scope, headings: I added summaries and, where needed, scopes, but there are already headings on all these tables. Am I missing something or was this just a copy/paste situation?

**Comment by maryfries** [comment]
L2P5 and L2P6 Tables are missing summary, scope, headings: We are using tables in a different way here. I added summaries, but headings are not appropriate, and therefore neither are scopes. Please have a look and let me know if we need to do more here.

**Comment by cycomachead** [comment]
I don't think they're an inaaprioate use of tables. The arrow is a little suspect, but I think that's probably OK. I'm not sure what the best accessibility thing is there, but I don't think it's a problem if that row doesn't have a good description. 

Otherwise, the fix is that the tables _should_ probably have some headings - at least a row heading for the binary value and one for the decimal representation. There is a semantic relationship between the data in the columns -- it's most clear in the hex/binary. The ones/sixteens columns are probably headings, and so are the ones for each N's place. 

The hex/binary table is also a little weird because it's kind of like 2 tables stuffed together and wouldn't read well at all for a screen reader - or someone who is red/blue color color blind. 
I think making the text description be 1 row, and having thicker borders around each byte would be more clear.

**Comment by maryfries** [comment]
Thanks. I believe all of these issues have now been resolved.
